Functions/Tests/CombineUnit/HTTPSCallableTests.swift

```diff
@@ -13,13 +13,15 @@
 // limitations under the License.
 
 import Foundation
+
 import Combine
-import XCTest
-import FirebaseFunctions
+import FirebaseFunctionsCombineSwift
```

Comment:
Somehow it compiled it CocoaPods without this, but only sometimes. There's some sort of Xcode module loading race condition that this fixes.